OsmAnd/src/net/osmand/plus/measurementtool/MeasurementToolFragment.java

```diff
@@ -123,16 +110,15 @@
 	private boolean gpxPointsAdded;
 	private ApplicationMode snapToRoadAppMode;
```

Comment:
private boolean gpxPointsAdded;
  private ApplicationMode snapToRoadAppMode;

This should be deleted